src/shared/components/case-viewer/viewer.routing.ts

```diff
@@ -23,5 +24,9 @@ export const viewerRouting: Routes = [
     {
       path: 'event/:eid/history',
       component: CaseHistoryComponent,
+    },
+    {
+      path: 'access-request',
```

Comment:
Do you think path should have the word `challenged` in it? Just a suggestion.

Reply:
@adnanakgun - could be an idea... although, I'm not sure if we're going to reuse this component for the "Specific Access" case - in which case, they _may_ want to use the same route... perhaps.